vms/platformvm/client.go

```diff
@@ -857,13 +857,12 @@ func (c *client) GetValidatorsAt(ctx context.Context, subnetID ids.ID, height ui
 }
 
 func (c *client) GetBlock(ctx context.Context, blockID ids.ID, options ...rpc.Option) ([]byte, error) {
-	response := &api.FormattedBlock{}
+	res := &api.FormattedBlock{}
```

Comment:
Better yet:
```go
res := &api.FormattedBlock{}
blockArgs := &api.GetBlockArgs{
    BlockID: blockID,
    EncodingL formatting.Hex,
}
err := c.requester.SendRequest(ctx, "platform.getBlock", blockArgs, res, options...)

if err != nil {
    return nil, err
}

return formatting.Decode(res.Encoding, res.Block)
```
IMO, multiline `if`s hurt readability by a large factor. Same with having to read `SendRequest` from left->right->down->right. Arguments should be top-to-bottom OR left-to-right... not both! 

Haha, just my (pedantic) opinions though and leaving them here since you're cleaning up anyway.

Reply:
We do this pattern in a few different places, will re-visit in another PR